cmake/packaging.cmake

```diff
@@ -117,7 +129,9 @@ if(EXISTS ${PROJECT_SOURCE_DIR}/build/logs/test.txt)
 endif()
 
 # Copy releasenotes.txt and appropriate ReadMe.txt (eg releasenotes/${PACKDIR}/ReadMe.txt)
-install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/releasenotes/${PACKDIR}/ReadMe.txt DESTINATION .)
-install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/releasenotes/releasenotes.txt      DESTINATION .)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/releasenotes/${PACKDIR}/ReadMe.txt ReadMe.txt       @ONLY)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/releasenotes/releasenotes.txt      releasenotes.txt @ONLY)
+install       (FILES  ${CMAKE_CURRENT_BINARY_DIR}/ReadMe.txt ${CMAKE_CURRENT_BINARY_DIR}/releasenotes.txt DESTINATION .)
+# file        (REMOVE ${CMAKE_CURRENT_BINARY_DIR}/releasenotes.txt ${CMAKE_CURRENT_BINARY_DIR}/ReadMe.txt)
```

Comment:
Hm, my uneducated eye says that there's a loop or a glob missing somewhere? Or is this working as this is?

Reply:
Well it's working!
`configure_file` copies a file and substitutes variables with the syntax `@VAR@`
`install(FILES ...)` copies the newly minted files into the bundle

Reply:
Looks fine to me. The only additional thing I would do is to remove the commented line 135. There is no need to remove those files that are generated in the build directory, since they are needed for the installation step.

Reply:
You are 100% correct _(as you always are)!_

I was puzzled when `file(REMOVE )...` caused an error, and I commented it off, meaning to delete it (which I forgot).  However, I know why it caused an error.  It's essential to leave the file in the build folder for the CPack to put it in the bundle.

I've already merged this PR, however I'm sure I'll be visiting this file again in Exiv2 v0.27.3 and I'll remove the `file (REMOVE ...` code at that time.

This is a good example of why using the `to-master` label isn't a good idea.  We want the smallest number of PRs being pushed to 'master'.  When Exiv2 v0.27.3 is ready, I'll bring 'master' into sync with my changes.